autoload/gutentags/ctags.vim

```diff
@@ -44,7 +44,7 @@ function! gutentags#ctags#generate(proj_dir, tags_file, write_mode) abort
 
     try
         " Build the command line.
-        let l:cmd = gutentags#get_execute_cmd() . s:runner_exe
+        let l:cmd = gutentags#get_execute_cmd() . '"' . s:runner_exe . '"'
```

Comment:
Don't do this. `shellescape()` exists for a reason, you'll run into more edge cases which were already solved by `shellescape()`.